config/routes.rb

```diff
@@ -7,6 +7,8 @@
     resources :reviews
   end
 
+  resources :reviews
```

Comment:
Your `reviews_controller` doesn't have all actions defined, and you definitely don't want all actions to be accessible as un-nested routes. You should restrict this to only the routes you want to be accessible in an un-nested way, and restrict your nested `reviews` resources to only those routes you want to be accessible in a nested way.